portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2778,6 +2797,11 @@ private void _checkFileEntriesByReviewDate(Date reviewDate)
 				fileEntry.getUserId(), latestFileVersion,
 				_buildEntryURL(latestFileVersion), "review",
 				new ServiceContext());
+
+			_notifyOwner(
+				fileEntry.getUserId(), latestFileVersion,
+				_buildEntryURL(latestFileVersion), "review",
```

Comment:
Don't use strings as enums. Either use an Enum or some private integer constans for "review" and "expire"; e.g.: _EMAIL_TYPE_REVIEW and _EMAIL_TYPE_EXPIRED.